static/script.js

```diff
@@ -1,4 +1,14 @@
+document.querySelectorAll(".emoji-button").forEach(function (button) {
+  button.addEventListener("webkitAnimationEnd", function() {
+    // reset animation
+    button.style.animationName = "";
+  });
+});
+
 function post_feedback(fb) {
+  var button = document.querySelector(".emoji-button." + fb);
```

Comment:
I'm actually surprised that `const` works in all current browser versions. I'm so used to transpilers and webpack that I totally missed out that browsers are pretty modern nowadays.